backend/routes/users.py

```diff
@@ -82,8 +88,8 @@ async def get_user_from_headers(headers: dict[str, str], db: AsyncSession):
 
 # profile endpoint must be declared first in order to function within
 # the /api/users context
-@router.get("/profile", response_model=schemas.UserRead)
-@router.get("/profile/", response_model=schemas.UserRead)
```

Comment:
why is the response model being removed here?

Reply:
I agree, as I was trying to fix react agent features this was done. But I am reverting it, as it is not applicable to local dev env.